plugin/web/extensions/sidebar/core/components/content/common/DatasetCard/Field/Fields/types.ts

```diff
@@ -62,17 +65,18 @@ export type Legend = FieldBase<"legend"> & {
   items?: LegendItem[];
 };
 
-// type Realtime = {
-//   type: "realtime";
-//   group?: string;
-//   updateInterval: number; // 1000 * 60 -> 1m
-// };
+type Realtime = FieldBase<"realtime"> & {
+  updateInterval?: number; // 1000 * 60 -> 1m
```

Comment:
You set this and the StyleCode's src in the component declaration. If these will always have values/need values please type them as necessary.

Reply:
Done

Reply:
c9e5706fc4bb3535ff518c320d648fb4f2ea4b1b